raiden/network/transport/matrix.py

```diff
@@ -1071,7 +1067,15 @@ def _maybe_invite_user(self, user: User):
             room.get_joined_members()
         if user.user_id not in room._members:
             self.log.debug('Inviting', user=user, room=room)
-            room.invite_user(user.user_id)
+            try:
+                room.invite_user(user.user_id)
+            except (json.JSONDecodeError, MatrixRequestError):
+                self.log.debug(
```

Comment:
This should be a warning - at least for a while until we're sure this works as intended